app/controllers/groups_controller.rb

```diff
@@ -29,6 +29,7 @@ def edit
   def create
     @group = Group.new(group_params)
     @group.organisation = current_user.organisation
+    @group.memberships.build(user: current_user, added_by: current_user)
```

Comment:
we add ourselves to the group we are creating, otherwise we won't be part of it and have any access